Extensions/video_downloader.js

```diff
@@ -0,0 +1,52 @@
+//* TITLE Video Downloader **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION	adds a download button to the video player **//
```

Comment:
the tabspace here is the xkit editor's fault but still needs to be removed (should be an actual space)
also proper capitalisation